rest/taskana-rest-spring/src/test/java/pro/taskana/rest/resource/ClassificationAssemblerTest.java

```diff
@@ -99,26 +99,27 @@ private void testLinks(ClassificationResource resource) {
-    Assert.assertEquals(classification.getCustom1(), classificationResource.custom1);
-    Assert.assertEquals(classification.getCustom2(), classificationResource.custom2);
-    Assert.assertEquals(classification.getCustom3(), classificationResource.custom3);
-    Assert.assertEquals(classification.getCustom4(), classificationResource.custom4);
-    Assert.assertEquals(classification.getCustom5(), classificationResource.custom5);
-    Assert.assertEquals(classification.getCustom6(), classificationResource.custom6);
-    Assert.assertEquals(classification.getCustom7(), classificationResource.custom7);
-    Assert.assertEquals(classification.getCustom8(), classificationResource.custom8);
-    Assert.assertEquals(classification.getParentId(), classificationResource.parentId);
-    Assert.assertEquals(classification.getType(), classificationResource.type);
-    Assert.assertEquals(classification.getPriority(), classificationResource.priority);
+        classification.getApplicationEntryPoint(),
+        classificationResource.getApplicationEntryPoint());
+    Assert.assertEquals(classification.getKey(), classificationResource.getKey());
+    Assert.assertEquals(classification.getDomain(), classificationResource.getDomain());
+    Assert.assertEquals(classification.getId(), classificationResource.getClassificationId());
+    Assert.assertEquals(classification.getDescription(), classificationResource.getDescription());
+    Assert.assertEquals(classification.getName(), classificationResource.getName());
+    Assert.assertEquals(classification.getServiceLevel(), classificationResource.getServiceLevel());
+    Assert.assertEquals(classification.getCategory(), classificationResource.getCategory());
+    Assert.assertEquals(classification.getCustom1(), classificationResource.getCustom1());
+    Assert.assertEquals(classification.getCustom2(), classificationResource.getCustom2());
+    Assert.assertEquals(classification.getCustom3(), classificationResource.getCustom3());
+    Assert.assertEquals(classification.getCustom4(), classificationResource.getCustom4());
+    Assert.assertEquals(classification.getCustom5(), classificationResource.getCustom5());
+    Assert.assertEquals(classification.getCustom6(), classificationResource.getCustom6());
+    Assert.assertEquals(classification.getCustom7(), classificationResource.getCustom7());
+    Assert.assertEquals(classification.getCustom8(), classificationResource.getCustom8());
+    Assert.assertEquals(classification.getParentId(), classificationResource.getParentId());
+    Assert.assertEquals(classification.getType(), classificationResource.getType());
+    Assert.assertEquals(classification.getPriority(), classificationResource.getPriority());
     Assert.assertEquals(
-        classification.getIsValidInDomain(), classificationResource.isValidInDomain);
+        classification.getIsValidInDomain(), classificationResource.getIsValidInDomain());
```

Comment:
Pls use AssertJ for this and future tests. If the dependency is not already added, add it like this to the corresponding
pom.xml: 	
	
<dependency>
			<groupId>org.assertj</groupId>
			<artifactId>assertj-core</artifactId>
			<version>${version.assertj}</version>
			<scope>test</scope>
		</dependency>

Then you can import the static assertthat...() methods